### DISCUSSION TITLE
Atomi Space Heater Support?

### DISCUSSION THREAD

**Discussion by nmajin** [ct-Open-Source/tuya-convert#1015] [Q&A]
I have an Atomi Space Heater which has an ESP chip (not 100%  if it is ESP8266, but going to assume it is). I have it setup via the Tuya app and all works great. I would like to NOT use the Tuya API with the Tuya integration (primarily because it is just a trial) and would like to use the Tuya Local setup.

Can I use Tuya Local out of the box or do I need to use Tuya Convert (can I even use it for this device?) to flash the firmware first? Again, assuming I am able to.

Any help would be appreciated.

**Comment by cooltech786** [comment]
I know this is an old post but Ive got the same heater and was wondering if you ever figured this out? Was it compatible with tuya convert?

**Reply by nmajin** [reply]
Long delay on a reply here, but just recently started messing with this and it is not compatible with tuya-convert. However, I did flash it by wiring up the esp chip to a usb-to-serial adapter and flashed ESPHome on it...the connectivity works, just figuring out the climate platform configuration needed to make it work proper.
